src/game/stage/background/zone_3.c

```diff
@@ -169,12 +169,14 @@ void StageBgUpdate_Zone3Acts12(s32 a, s32 b)
 
             while (r6 >= cursor[i * 3]) {
                 if (++i >= ARRAY_COUNT(gUnknown_080D5B20)) {
-                    goto _0801CCA8;
+                    break;
+                }
+                {
```

Comment:
Does this need to be scoped?

Reply:
Keep in mind for both chnages in zone_3.c that they're still non-matching.
Changed that bit because of some tests I did and compiler errors I got because of the goto.

Reply:
Ah I didn't notice!